admin/class-insert-rule-data.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Inserts rule data about a post to the database
+ *
+ * @since 2.0.0
+ *
+ * @package Accessibility_Checker
+ */
+
+namespace EDAC\Admin;
+
+/**
+ * Class for inserting rule data into the database
+ *
+ * @since 2.0.0
+ */
+class Insert_Rule_Data {
+
+	/**
+	 * Insert rule data into database
+	 *
+	 * @since 2.0.0
+	 *
+	 * @param object $post     The post object.
```

Comment:
Update this to:
`@param object $post     The post object. Must have a valid ID property.`